cucumber/kubernetes/features/support/world.rb

```diff
@@ -91,15 +93,17 @@ def find_matching_pod(label_selector)
   def detect_request_ip objectid
     expect(objectid).to match(/^([\w-])+\/([\w-])+$/)
     controller_type, id = objectid.split('/')
-    controller = k8s_object_lookup.find_object_by_name controller_type, id, namespace
+    controller          = k8s_object_lookup.find_object_by_name controller_type, id, namespace
```

Comment:
I don't think it's really necessary to align this equals. I find such alignment to only be useful for long lists of assignments with single-value right-hand sides.

Reply:
i'm ok with both ways. It was the auto-indent. Changing it back to original